src/administrator/components/com_kunena/controllers/categories.php

```diff
@@ -441,11 +441,11 @@ protected function _save()
 			}
 
 			$success = $category->save();
-			$aliases = explode(',', JRequest::getVar('aliases_all'));
+			$aliases = explode(',', $app->input->getString('aliases_all'));
 
 			if ($aliases)
 			{
-				$aliases = array_diff($aliases, JRequest::getVar('aliases', array(), 'post', 'array'));
+				$aliases = array_diff($aliases, $app->input->post->getArray(array('aliases' => '')));
```

Comment:
with this change all options are going to be removed.

test:
adding some aliases
when you have 4 aliases, then select only the last 3, then click save. Result. only last one is there left